plugins/Makefile

```diff
@@ -225,16 +225,26 @@ plugins/recover:  common/gossmap.o common/fp16.o $(PLUGIN_RECOVER_OBJS) $(PLUGIN
 
 # This covers all the low-level list RPCs which return simple arrays
 SQL_LISTRPCS := listchannels listforwards listhtlcs listinvoices listnodes listoffers listpeers listpeerchannels listclosedchannels listtransactions listsendpays bkpr-listaccountevents bkpr-listincome
-SQL_LISTRPCS_SCHEMAS := $(foreach l,$(SQL_LISTRPCS),doc/schemas/$l.schema.json)
-# We squeeze:
-#  descriptions (we don't need)
-#  fields with no members (we don't need)
-#  whitespace
-# We can't simply *remove* fields, since the extra comma left over can
-# make invalid JSON. Grr!
-# But these simple removals drop us from 100k to 29k.
+SQL_LISTRPCS_SCHEMAS := $(foreach l,$(SQL_LISTRPCS),doc/schemas/lightning-$l.json)
+
+# RES_JSON is generated by:
+# 1: deleting description array (used for field details) but keeping description keys
+# 2: deleting additionalProperties, required, enum, maxLength, minLength, pre_return_value_notes, post_return_value_notes, anyOf, oneOf, hidden, untyped, default, comment, added, maximum, minimum, pattern, format
+# (We only need [field].type, [field].name, [field].deprecated, [field].allOf, [field].items.allOf and allOf from response.properties)
+# 3: deleting empty objects from .response.properties
```

Comment:
Nice!  This actually reduces the header file from 44081 bytes to 18477!

Reply:
Added it into the comments too.